_includes/app/txn-sample.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/lib/pq"
+)
+
+func transferFunds(tx *sql.Tx, from int, to int, amount float64) error {
+	// Read the balance.
+	var fromBalance float64
+	if err := tx.QueryRow(
+		"SELECT balance FROM accounts WHERE id = $1", from).Scan(&fromBalance); err != nil {
+		return err
+	}
+
+	if fromBalance < amount {
+		return fmt.Errorf("insufficient funds")
+	}
+
+	// Transfer $100 to the other account.
+	if _, err := tx.Exec(
+		"UPDATE accounts SET balance = balance - $1 WHERE id = $2", amount, from); err != nil {
+		return err
+	}
+	if _, err := tx.Exec(
+		"UPDATE accounts SET balance = balance + $1 WHERE id = $2", amount, to); err != nil {
+		return err
+	}
+	return nil
+}
+
+// txnWrapper runs fn inside a transaction and retries it as needed.
+func txnWrapper(db *sql.DB, database string, fn func(*sql.Tx) error) error {
+	// Start a transaction.
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	// Specify that we intend to retry this txn in case of CockroachDB retriable
```

Comment:
I like that idea, @spencerkimball. I'm starting to think it might be more helpful to show this sequence for each language:
1. Create db as root user.
2. Grant some user permissions on that db.
3. Show that user connecting and running simple statements from client (create table, insert, select). 
4. Show transaction for transferring units between accounts. Start by showing and explaining necessity of re-usable `txnWrapper`. Then show it used in transfer transaction. 

This would mean we wouldn't have two distinct examples per language but rather one example that guides you through. Or would you rather keep the basic and transaction samples separate?